Source/WebCore/platform/graphics/DoubleSize.h

```diff
@@ -25,79 +25,36 @@
 
 #pragma once
 
-#if USE(CG)
-#import <CoreGraphics/CGGeometry.h>
-#endif
+#include "IntSize.h"
+#include <wtf/MathExtras.h>
 
-namespace WebKit {
-
-struct DoublePoint {
 #if USE(CG)
-    DoublePoint(const CGPoint& point)
-        : DoublePoint(point.x, point.y)
-    {
-    }
-
-    CGPoint toCG() const
-    {
-        return { static_cast<CGFloat>(x), static_cast<CGFloat>(y) };
-    }
+typedef struct CGSize CGSize;
 #endif
 
-    DoublePoint(double x, double y)
-        : x(x)
-        , y(y)
-    {
-    }
+namespace WebCore {
 
-    double x { 0 };
-    double y { 0 };
-};
+class DoubleSize {
```

Comment:
This PR is quite large as-is, so I would recommend splitting up any changes involving the moving of types from WebKit -> WebCore into a separate, first PR.